test_shaders.py

```diff
@@ -1,5 +1,5 @@
-#!/usr/bin/env python3
-
+#!/usr/bin/python
```

Comment:
Also, CMakeLists.txt would also have to be updated with python 2 checks instead if both python2/3 will work.